.env.template

```diff
@@ -6,4 +6,4 @@ CLIENT_PUBLIC_KEYS_DIRECTORY=
 SERVER_PUBLIC_KEY=
 SERVER_PRIVATE_KEY=
 SESSION_LIFETIME_MINUTES=
-REDIRECT_URL_TO_LOUIS_FRONTEND=
+REDIRECT_URL_TO_MEMBRANE_FRONTEND=
```

Comment:
it would be a lot clearer if your prefixed all your environment variables with MEMBRANE_ instead of embedding in the middle in some, but not all configuration strings.

Makes the job easier when devops has to juggle tens if not hundreds of environment variables at the organization level.

Reply:
This is already done.